File: src/java/magmac/app/stage/after/EmptyAfterAll.java
package magmac.app.stage.after;

import magmac.app.compile.node.Node;
import magmac.app.stage.unit.UnitSet;

public class EmptyAfterAll implements AfterAll {
    @Override
    public UnitSet<Node> afterAll(UnitSet<Node> roots) {
        return roots;
    }
}

File: src/web/magmac/app/stage/after/EmptyAfterAll.ts
import { Node } from "../../../../magmac/app/compile/node/Node";
import { UnitSet } from "../../../../magmac/app/stage/unit/UnitSet";
export class EmptyAfterAll {
	afterAll(roots : UnitSet<Node>) : UnitSet<Node> {return 0;;}
}
